src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,13 @@
-export const TodoList = () => {};
+/* eslint-disable max-len */
+import { TodoInfo } from '../TodoInfo';
+import { ToDosWithUserProps } from '../types';
+
+export const TodoList: React.FC<ToDosWithUserProps> = ({ todos }) => {
+  return (
```

Comment:
if you don't have any logic inside your component and return only jsx then you can use it without `return` like 

```
export const TodoList: React.FC<ToDosWithUserProps> = ({ todos }) => (
  <div>
    <section className="TodoList">
      {todos.map(todo => (
        <TodoInfo key={todo.id} todo={todo} data-id={todo.id} />
      ))}
    </section>
  </div>
);
```

check and update all your components